benchmarks/compile_protos.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Consider moving this file to `benchmarks/tool`

Reply:
Done.